custom_model_runner/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+#### [1.4.9] 
+##### Changes
+- `transform` mode now takes and returns both X and y, although passing y is optional.
```

Comment:
the changelogs don't have anyhthing about the endpoint in them.

Reply:
damn, sure enough